### DISCUSSION TITLE
Plural form is lost in translation

### DISCUSSION THREAD

**Discussion by xperseguers** [WeblateOrg/weblate#7053] [Q&A]
Hi,

I created a source `.po` file like that:

```
# Group of attributes for member's details
msgid "Person.Group.Sponsors"
msgid_plural ""
msgstr[0] "Sponsor"
msgstr[1] "Sponsors"
```

And this is properly shown in Frontend:
![sponsor](https://user-images.githubusercontent.com/646805/147947082-f2489692-da43-4af5-ac2a-bfa632c8b652.png)

However when I look at the generated `.po` files in `~/data/vcs/...` after committing back, the plural form is completely lost:

```
msgid "Person.Group.Sponsors"
msgstr "Parrain"
```

Since I'm generating the source English file, I can easily adapt the generated format, I guess that somehow I need to put something for `msgid_plural`? What I expect "somehow" in the generated French `.po` file is to get back the `msgstr[0]` and `msgstr[1]` plural forms. And naturally further plural forms in case the language has more than 2 plural forms.

Can you please help?

**Comment by nijel** [comment]
Most likely, the plurals were not tested in monolingual PO files, so there could be a bug in saving them...

**Reply by xperseguers** [reply]
Do you mean I shall click the link to transform this discussion as a ticket? Or is there something else I may try before?

**Reply by xperseguers** [reply]
#7061
